<launch>
  <arg name="roi" default="20.0" />
  <arg name="time_window" default="0.2" />
  <arg name="global_frame" default="odom" />
  <arg name="sensor_frame" default="velodyne" />
  <arg name="xy_scale" default="2" />
  <arg name="scale_axis" default='y' />

  <node pkg="nodelet" type="nodelet" name="pointcloud_lidar_processing"  args="manager" output="screen">
    <param name="roi" value="$(arg roi)" />
    <param name="time_window" value="$(arg time_window)" />
    <param name="global_frame" value="$(arg global_frame)" />
    <param name="sensor_frame" value="$(arg sensor_frame)" />
    <param name="xy_scale" value="$(arg xy_scale)" />
    <param name="scale_axis" value="$(arg scale_axis)" />
  </node>

  <node pkg="nodelet" type="nodelet" name="pointcloud_nodelet" args="load gr_pointcloud_processing/PointCloudProcessor pointcloud_lidar_processing" output="screen"/>
  <!--node pkg="tf2_ros" type="static_transform_publisher" name="tf_broadcaster_base_lidar" args="0 0 0 1.5755 0 0  base_link velodyne" /-->
  <node pkg="tf2_ros" type="static_transform_publisher" name="tf_broadcaster_base_lidar" args="0 0 0 0 0 0  base_link velodyne" />
</launch>
